### DISCUSSION TITLE
[Office Hours] Xcode won't open after resolving merge conflicts.

### DISCUSSION THREAD

**Discussion by yanavlall** [CS342/.github#64] [Office Hours] [answered]
### Related Discussion

My code to display and play a video was working fine but after resolving merge conflicts, i just get a white screen. 

### Reproduction

-

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment]
@yanavlall  Has your question already been resolved? If so, could you share with the class how you resolved the issue?

**Comment by yanavlall** [comment, marked as answer]
Yes, it has been resolved. I went to GitHub Desktop and reverted the changes that had initially led to XCode not opening. Then, to fix the white screen issue for my video, I realised I had not selected the "copy if needed" option when dragon dropping my video into XCode which meant that while my code would pass the build test on my computer as I had the video downloaded, it wouldn't on GitHub because the video wasn't available there (the path was not what it should have been) until I selected the "copy if needed" option. So, I did that and it works now.

**Reply by PSchmiedmayer** [reply]
Thank you for sharing the solution @yanavlall! A lot of students have similar issues, this might help them to resolve it; greatly appreciated!
